src/examples/example.yml

```diff
@@ -15,6 +15,7 @@ usage:
         - run: npm test
         - qlty/coverage_publish:
             files: coverage/lcov.info, coverage/lcov_2.info # Comma separated list of coverage files to upload
+            report_format: lcov
```

Comment:
The CLI infers files with a `.info` extension to be LCOV format, so this should not be needed

Reply:
Removed